### PR TITLE
Update the Copy for all Modals in the Admin Member Detail View

### PR DESCRIPTION
**Pull request by juliannatetreault**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
This PR updates the copy for the modals throughout the Admin Member Detail view per [this](https://www.figma.com/file/sZBBadfB9OqO6wpfBdXjRf/(copy-edited)-member-detail-view?node-id=0%3A1) figma file. 

## Related Tickets & Documents
- Closes Issue #16507 

## QA Instructions, Screenshots, Recordings
Please ensure that there are no spelling errors. You can also compare these copy changes against the figma file. Finally, ensure that all checks pass. ♻️ 

#### After:
Screen Shot 2022-02-25 at 11 36 07 AM

### UI accessibility concerns?
N/A

## Added/updated tests?

- [x] Yes
- [ ] No, and this is why: 
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://developers.forem.com) or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] This PR changes the Forem platform and our documentation needs to be
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
- [ ] I've updated the README or added inline documentation
- [ ] I've added an entry to
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [x] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [ ] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] Are there any post deployment tasks we need to perform?
N/A

## [optional] What gif best describes this PR or how it makes you feel?

"Boom! Accomplished."

### CODE REVIEW

File: cypress/integration/seededFlows/adminFlows/users/manageUserOptions.spec.js [hunk 1]
```diff
@@ -67,22 +67,22 @@ describe('Manage User Options', () => {
...
 
     it(`should banish a user for spam`, () => {
       openUserOptions(() => {
-        cy.findByRole('button', { name: 'Banish for spam' }).click();
+        cy.findByRole('button', { name: 'Banish user' }).click();
       });
 
       cy.getModal().within(() => {
-        cy.findByRole('button', { name: 'Banish User for spam' }).click();
+        cy.findByRole('button', { name: 'Banish Trusted User 1 :/' }).click();
```
Comment (juliannatetreault) [inline cypress/integration/seededFlows/adminFlows/users/manageUserOptions.spec.js:85]: This is failing, as Cypress can't find this button. That said, I've tried a handful of things to try and get this to work, but for some reason it's not picking up this "banish" button's text. Any ideas on how to resolve this? 🤔

Comment (aitchiss) [reply cypress/integration/seededFlows/adminFlows/users/manageUserOptions.spec.js:85]:
```suggestion
        cy.findByRole('button', { name: 'Banish Trusted User 1 \:/' }).click();
```
I haven't tried this locally, but the output from the test failure lists this as the available button name ^ (just missing the backslash)

File: app/views/admin/users/show/profile/actions/_export.html.erb [hunk 2]
```diff
@@ -1,19 +1,15 @@
 <div id="export-data">
   <div class="flex flex-col gap-4">
     <p>
-      You can export a user's data here. Currently we only support exporting a user's posts and comments.
+      Choose betweem sending the exported data to either your Admin contact email, <%= ForemInstance.email %>,
+      or to <%= @user.name %>'s <a href="<%= @user.email %>">user email</a> directly.
```
Comment (aitchiss) [inline app/views/admin/users/show/profile/actions/_export.html.erb:5]:
```suggestion
      or to <%= @user.name %>'s <a href="mailto:<%= @user.email %>">user email</a> directly.
```

At the moment this is directing to e.g. `http://localhost:3000/admin/users/lidia+collier@oberbrunner.info`, which doesn't exist

**Review by aitchiss — APPROVED** [review]
Just a small suggestion to fix an email link, but otherwise this looks great 🚀 

Let me know if my suggestion on the Cypress test doesn't work and you need another pair of eyes on it 👀

File: app/views/admin/users/show/profile/actions/_export.html.erb (hunk 2 above)
Comment (juliannatetreault) [reply app/views/admin/users/show/profile/actions/_export.html.erb]: Great catch; thanks!!

**Comment by juliannatetreault** [comment]
> Just a small suggestion to fix an email link, but otherwise this looks great 🚀
>
> Let me know if my suggestion on the Cypress test doesn't work and you need another pair of eyes on it 👀

Thanks so much for the review, @aitchiss! I actually could use a second pair of eyes on the cypress failure, as I had tried your suggested fix last week and again today to no avail and I'm feeling out of ideas for how to target the button properly. 😕

File: cypress/integration/seededFlows/adminFlows/users/manageUserOptions.spec.js (hunk 1 above)
Comment (aitchiss) [reply cypress/integration/seededFlows/adminFlows/users/manageUserOptions.spec.js:85]:
```suggestion
        cy.findByRole('button', { name: 'Banish Trusted User 1 \\:/' }).click();
```

OK so what's happening is `\` is escaping the next character.. but `:` doesn't need escaped, so the linter removes it and we're left with `User 1 :/` when what we need is `User 1 \:/` 🙈 

If we escape the backslash like this it works as expected. Smh. What a weird one 😄  I've tested this locally and it all passed OK

File: app/views/admin/users/show/profile/actions/_banish.html.erb [hunk 3]
```diff
@@ -1,20 +1,21 @@
 <div id="banish-for-spam">
   <div class="flex flex-col gap-4">
     <% if @user.comments.where("created_at < ?", 100.days.ago).empty? && @user.created_at < 100.days.ago %>
-      <p><strong>This is extremely destructive.</strong> Banishing will delete all the user's existing content and change their username to <em>@spam_###</em>.</p>
-      <p><strong>Do not do this lightly.</strong></p>
-      <%= form_for(@user, url: banish_admin_user_path(@user), html: { method: :post, onsubmit: "return confirm('Are you sure? This is extremely destructive and irreversible. Banishing will delete all articles and turn their username into @spam_###')", id: nil }) do %>
+      <p>This action is irreversible.</p>
```
Comment (jeremyf) [inline app/views/admin/users/show/profile/actions/_banish.html.erb:4]: Reading notes: it looks like we have two chunks of HTML that are identical.  We have chosen to separate them by an `if` then `elsif` to highlight the purpose.

Non-blocking recommendation: make a helper method that answers the question "should we render the banish a user section?"  We could then use line 3 and line 10 as the clauses in our function.

**Review by jeremyf — APPROVED** [review]

File: cypress/integration/seededFlows/adminFlows/users/manageUserOptions.spec.js (hunk 1 above)
Comment (juliannatetreault) [reply cypress/integration/seededFlows/adminFlows/users/manageUserOptions.spec.js]: Oh my gosh! 🙈  Thank you sooooo much for the help, @aitchiss! 🙌

File: app/views/admin/users/show/profile/actions/_banish.html.erb (hunk 3 above)
Comment (juliannatetreault) [reply app/views/admin/users/show/profile/actions/_banish.html.erb:4]: I'll refactor this in a follow-up PR, @jeremyf 😊

**Pull request merged**